sentry-android-core/src/main/java/io/sentry/android/core/SentryFrameMetrics.java

```diff
@@ -73,16 +62,14 @@ public long getFrozenFrameDelayNanos() {
   }
 
   public int getTotalFrameCount() {
-    return normalFrameCount + slowFrameCount + frozenFrameCount;
+    return slowFrameCount + frozenFrameCount;
```

Comment:
but the total frame count would be incorrect now, why are we removing it?

Reply:
we are still calculating the normal frames [based on the span duration](https://github.com/getsentry/sentry-java/pull/3782/files#diff-8fb51c60f3cd9eba198cd0702cf5b06a1b4e0ff676dccaa371840a44a5a5bd8eL268).
Actually, previously we were interpolating the number of frames by removing the fast frames duration, too, which could lead to having more total frames than expected. Now it should be more accurate.
E.g. a frame take 2ms to render in a span of 10ms.
Previously we were adding the frames that we were interpolating based on the 8ms (10-2) which was logically wrong. 

Does it make sense?

Reply:
> If the list doesn't contain the normal frames anymore, the duration may be considered as a delayed frame. E.g. if a span takes 1000ms and only normal frames happen, the list will be empty and it will be considered a frame delay

That part is skipped if the list is empty, and the whole span is considered without delay

> the method is called getTotalFrameCount but it only returns the sum of slow+frozen

Ok 😅  I'll rename it to `getSlowFrozenFrameCount`